webclient/copy_shared_files.ps1

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+robocopy /E ../common/pb/. ./public/pb/
```

Comment:
Lets keep this file Mac-friendly.

Reply:
I left it Mac-Friendly with `copy_shared_files.sh` but just adding that file for windows.